mix.lock

```diff
@@ -1,15 +1,14 @@
-  "erlex": {:hex, :erlex, "0.2.6", "c7987d15e899c7a2f34f5420d2a2ea0d659682c06ac607572df55a43753aa12e", [:mix], [], "hexpm", "2ed2e25711feb44d52b17d2780eabf998452f6efda104877a3881c2f8c0c0c75"},
-  "ex_doc": {:hex, :ex_doc, "0.31.2", "8b06d0a5ac69e1a54df35519c951f1f44a7b7ca9a5bb7a260cd8a174d6322ece", [:mix], [{:earmark_parser, "~> 1.4.39", [hex: :earmark_parser, repo: "hexpm", optional: false]}, {:makeup_c, ">= 0.1.1", [hex: :makeup_c, repo: "hexpm", optional: true]}, {:makeup_elixir, "~> 0.14", [hex: :makeup_elixir, repo: "hexpm", optional: false]}, {:makeup_erlang, "~> 0.1", [hex: :makeup_erlang, repo: "hexpm", optional: false]}], "hexpm", "317346c14febaba9ca40fd97b5b5919f7751fb85d399cc8e7e8872049f37e0af"},
-  "file_system": {:hex, :file_system, "1.0.0", "b689cc7dcee665f774de94b5a832e578bd7963c8e637ef940cd44327db7de2cd", [:mix], [], "hexpm", "6752092d66aec5a10e662aefeed8ddb9531d79db0bc145bb8c40325ca1d8536d"},
-  "jason": {:hex, :jason, "1.4.1", "af1504e35f629ddcdd6addb3513c3853991f694921b1b9368b0bd32beb9f1b63", [:mix], [{:decimal, "~> 1.0 or ~> 2.0", [hex: :decimal, repo: "hexpm", optional: true]}], "hexpm", "fbb01ecdfd565b56261302f7e1fcc27c4fb8f32d56eab74db621fc154604a7a1"},
-  "makeup": {:hex, :makeup, "1.1.1", "fa0bc768698053b2b3869fa8a62616501ff9d11a562f3ce39580d60860c3a55e", [:mix], [{:nimble_parsec, "~> 1.2.2 or ~> 1.3", [hex: :nimble_parsec, repo: "hexpm", optional: false]}], "hexpm", "5dc62fbdd0de44de194898b6710692490be74baa02d9d108bc29f007783b0b48"},
-  "makeup_elixir": {:hex, :makeup_elixir, "0.16.2", "627e84b8e8bf22e60a2579dad15067c755531fea049ae26ef1020cad58fe9578", [:mix], [{:makeup, "~> 1.0", [hex: :makeup, repo: "hexpm", optional: false]}, {:nimble_parsec, "~> 1.2.3 or ~> 1.3", [hex: :nimble_parsec, repo: "hexpm", optional: false]}], "hexpm", "41193978704763f6bbe6cc2758b84909e62984c7752b3784bd3c218bb341706b"},
-  "makeup_erlang": {:hex, :makeup_erlang, "0.1.5", "e0ff5a7c708dda34311f7522a8758e23bfcd7d8d8068dc312b5eb41c6fd76eba", [:mix], [{:makeup, "~> 1.0", [hex: :makeup, repo: "hexpm", optional: false]}], "hexpm", "94d2e986428585a21516d7d7149781480013c56e30c6a233534bedf38867a59a"},
+  "credo": {:hex, :credo, "1.7.10", "6e64fe59be8da5e30a1b96273b247b5cf1cc9e336b5fd66302a64b25749ad44d", [:mix], [{:bunt, "~> 0.2.1 or ~> 1.0", [hex: :bunt, repo: "hexpm", optional: false]}, {:file_system, "~> 0.2 or ~> 1.0", [hex: :file_system, repo: "hexpm", optional: false]}, {:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: false]}], "hexpm", "71fbc9a6b8be21d993deca85bf151df023a3097b01e09a2809d460348561d8cd"},
+  "dialyxir": {:hex, :dialyxir, "1.4.4", "fb3ce8741edeaea59c9ae84d5cec75da00fa89fe401c72d6e047d11a61f65f70", [:mix], [{:erlex, ">= 0.2.7", [hex: :erlex, repo: "hexpm", optional: false]}], "hexpm", "cd6111e8017ccd563e65621a4d9a4a1c5cd333df30cebc7face8029cacb4eff6"},
```

Comment:
that version of 1.4.4 is problematic with OTP27. let s hold off till 1.4.5 is out

Reply:
Doesn't seem to affect Hammer.

```console
$ mix dialyzer
Compiling 2 files (.ex)
Finding suitable PLTs
Checking PLT...
[:bunt, :compiler, :credo, :crypto, :earmark_parser, :eex, :elixir, :ex_doc, :ex_unit, :file_system, :hammer, :inets, :jason, :kernel, :logger, :makeup, :makeup_elixir, :makeup_erlang, :nimble_parsec, :stdlib]
Looking up modules in dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt
Looking up modules in dialyxir_erlang-27.1.2_elixir-1.17.3.plt
Looking up modules in dialyxir_erlang-27.1.2.plt
Finding applications for dialyxir_erlang-27.1.2.plt
Finding modules for dialyxir_erlang-27.1.2.plt
Creating dialyxir_erlang-27.1.2.plt
Looking up modules in dialyxir_erlang-27.1.2.plt
Removing 3 modules from dialyxir_erlang-27.1.2.plt
Checking 19 modules in dialyxir_erlang-27.1.2.plt
Adding 196 modules to dialyxir_erlang-27.1.2.plt
done in 0m17.77s
Finding applications for dialyxir_erlang-27.1.2_elixir-1.17.3.plt
Finding modules for dialyxir_erlang-27.1.2_elixir-1.17.3.plt
Copying dialyxir_erlang-27.1.2.plt to dialyxir_erlang-27.1.2_elixir-1.17.3.plt
Looking up modules in dialyxir_erlang-27.1.2_elixir-1.17.3.plt
Checking 215 modules in dialyxir_erlang-27.1.2_elixir-1.17.3.plt
Adding 258 modules to dialyxir_erlang-27.1.2_elixir-1.17.3.plt
done in 0m10.86s
Finding applications for dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt
Finding modules for dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt
Copying dialyxir_erlang-27.1.2_elixir-1.17.3.plt to dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt
Looking up modules in dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt
Checking 473 modules in dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt
Adding 595 modules to dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt
done in 0m47.39s
No :ignore_warnings opt specified in mix.exs and default does not exist.

Starting Dialyzer
[
  check_plt: false,
  init_plt: ~c"/Users/x/Developer/hammer/_build/dev/dialyxir_erlang-27.1.2_elixir-1.17.3_deps-dev.plt",
  files: [~c"/Users/x/Developer/hammer/_build/dev/lib/hammer/ebin/Elixir.Hammer.ETS.beam",
   ~c"/Users/x/Developer/hammer/_build/dev/lib/hammer/ebin/Elixir.Hammer.beam"],
  warnings: [:unknown]
]
Total errors: 0, Skipped: 0, Unnecessary Skips: 0
done in 0m1.51s
done (passed successfully)
```